docs/simulators.rst

```diff
@@ -31,6 +31,19 @@ You can install it with:
 
 Scenic supports both 2D and 3D rendering modes for MetaDrive simulations.
 
+.. note::
+
+   MetaDrive **0.4.3** (the current PyPI release) does **not** support Python 3.12/3.13.
+   It also has known issues on macOS Apple Silicon (M-series) with 3D rendering and a
+   braking issue where vehicles may not come to a complete stop.
+
+   To use Python 3.12+ **and** get the fixes for the macOS/braking issues, install
+   MetaDrive from the GitHub repo:
+
+   .. code-block:: console
+
+      python -m pip install "metadrive-simulator @ git+https://github.com/metadriverse/metadrive.git@main"
```

Comment:
I think we need to add `sumolib` here too, since if `scenic[metadrive]` failed to install users may not have `sumolib`, which we need.